forwards_paper.tex

```diff
@@ -656,6 +661,26 @@ \subsection*{Recording the pedigree in forwards time}
 ancestors in the current population, and so the entire path tracing back to
 its common ancestor with 27 is redundant.
 
+\paragraph{Essential steps in tree sequence recording}
```

Comment:
Question: (a) am I missing anything, and more importantly, (b) do we want to do this here?  We talked about putting this in the msprime documentation rather than the paper, so: maybe not.

Reply:
I think this is the right place for it, and I think it's a good high-level summary of what people need to do.